source/databases-collections.txt

```diff
@@ -75,6 +77,24 @@ appear in that collection's documents. If a field exists in all
 documents and its type is consistent throughout the collection, |vsce|
 displays an icon indicating that field's data type.
 
+Indexes
+```````
+
+Your collections's indexes appear under the :guilabel:`Indexes` heading.
+When you expand an index, each index key appears with an icon designating
+its type. Index key types include:
+
+- Ascending
+- Descending
+- Geospatial (2d, 2dsphere, geoHaystack)
+- Text
+- Hashed
+
```

Comment:
Note: Atlas Search indexes aren't listed. I put a comment in the ticket to inquire.